index.bs

```diff
@@ -1329,23 +1340,39 @@ Removing AB or TAG members</h5>
 	Individual participants of the [=Advisory Board=] or the [=Technical Architecture Group=]
 	can be <dfn export lt="remove|removal">removed</dfn> from those groups
 	if they are found by their peers
-	to be grossly neglecting their duties,
-	or to be acting in a way that seriously hampers the group's ability to function.
+	to be grossly neglecting their duties
+	or acting in a way that seriously hampers the group's ability to function;
+	or if they are violating policies or other standards of behavior
+	(such as applicable laws or the <a href="https://www.w3.org/policies/code-of-conduct/#Reporting">Code of Conduct</a>)
```

Comment:
```suggestion
	(such as applicable laws or the <a href="https://www.w3.org/policies/code-of-conduct/">Code of Conduct</a>)
```

Reply:
This parenthetical is expanding on "policies or other standards of behavior", so linking to the document as a whole, and not specifically to the reporting section, is appropriate.